### ISSUE TITLE
Space after period in long LOT and LOF

### ISSUE THREAD

**Issue by piyanatk** [GarenSidonius/ASU-Dissertation-Template#12]
When you have a long LOT and LOF with 10 or more entries, the current setting will leave no gap between the period and the list/figure caption. I use `1.5\cftchapternumwidth` and it seems to work well.
```
\setlength{\cftfigurenumwidth}%         % Figure has the same 'level' as 'chapter' in the
  {1.5\cftchapternumwidth}              %   figure list, so make the number spacing the
                                        %   same as for chapters
\setlength{\cfttablenumwidth}%          % Table has the same 'level' as 'chapter' in the
  {1.5\cftchapternumwidth}              %   table list, so make the number spacing the
                                        %   same as for chapters
```

**Comment by GarenSidonius** [comment]
Do you happen to have a sample showing this problem? That way I can confirm that the proposed change fixes the problem. 

Have you also check the ASU formatting guidelines to make sure that this change conforms to the guidelines? I can check in case you haven't yet, but I would want to make sure that any formatting changes conform to the guidelines.

**Comment by piyanatk** [comment]
Here are screen shots before and after adding 1.5

Before
![before](https://user-images.githubusercontent.com/2381361/55248128-60deb400-5206-11e9-8d03-cdf4abb4a7eb.jpg)

After
![after](https://user-images.githubusercontent.com/2381361/55248142-69cf8580-5206-11e9-9840-a18bc35ff73a.jpg)

ASU formatting guideline only have rules for margins and header, so I think this is okay as long as you do not put excessive space in between the numbers and captions.

**Comment by GarenSidonius** [comment]
I've been thinking about this one a bit more, and I'm less sure about making this change. For someone with many tables and figures, the change is great, but for someone with fewer, the change leaves a large white space between the period and the start of the list/figure title. And I'm hesitant to make a change that is dissatisfying for some users and could subsequently lead to the opposite ticket ("please reduce white space between period and caption title"). 

Ideally, the change would take into account the total number of lists and figures in the document and then set the indent accordingly. I haven't looked into how to do that, yet, but if we make a change, I think that's what the change should be because it's the most general, user-friendly way to set the indent. 

What do you think? Do you agree/disagree?

**Comment by piyanatk** [comment]
Sorry for my late reply. Why don't we provide toggle for default and long list and use that to set different space, similar to how you set up natbib/biblatex option?

**Comment by GarenSidonius** [comment]
Thanks for your feedback. To me, creating a toggle is equivalent to the current situation where users might need to manually update something in the template in order to customize the template for its contents. Whether the user changes an indent value or changes a toggle is more or less the same. 

This attribute of the document (indent between table/figure number and table/figure caption) is a function of what's in the document, not a user preference. The indent is like page numbering; LaTeX automatically numbers each page correctly based on the contents of the document without the user having to do anything. In contrast, selecting a bibliography package is a user preference. 

So from my perspective, the best update would be to set the indent correctly automatically based on the content of the document to make it behave like other kinds of formatting that should happen automatically based on the content of the document.

**Comment by piyanatk** [comment]
In that case, I think the template should provide users with some information on how to adjust this as a troubleshoot until the automatic adjustment is added.

**Comment by GarenSidonius** [comment]
This issue can serve as that documentation until it's properly resolved.

**Comment by GarenSidonius** [comment]
This issue is resolved in 08f9b4b8ba762b5c6c2be85984311b2d5f96cd27.
